ush/python/pygfs/task/snow_analysis.py

```diff
@@ -50,12 +50,14 @@ def __init__(self, config: Dict[str, Any]):
         _res = int(self.task_config['CASE'][1:])
 
         # if 00z, do SCF preprocessing
-        _ims_file = os.path.join(self.task_config.COMIN_OBS, f'{self.task_config.OPREFIX}imssnow96.asc')
-        logger.info(f"Checking for IMS file: {_ims_file}")
-        if self.task_config.cyc == 0 and os.path.exists(_ims_file):
-            _DO_IMS_SCF = True
-        else:
-            _DO_IMS_SCF = False
+        _imsnc_file = os.path.join(self.task_config.COMIN_OBS, f'{self.task_config.OPREFIX}imssnow96.nc')
```

Comment:
Would it make sense to define a variable indicating whether we're expecting netcdf or ascii? Then we can use that as the extension? That would be easier to follow (particularly when theres a problem with the file).

Reply:
The C++ code can automatically handle both NetCDF and ASCII data without depending on the filename extension. It will also provide useful information if there is an issue with the file. Therefore, I updated the file extension from `.nc`/`.asc` to `.dat`. Wait for @CoryMartin-NOAA’s feedback.